lib/recipes/ap-physics-2e/locales/en.yml

```diff
@@ -4,5 +4,5 @@ en:
     glossary: Glossary
     section-summary: Section Summary
     conceptual-questions: Conceptual Questions
-    problems-exercises: Problems & Exercises
+    problems-exercises: Problems &amp; Exercises
```

Comment:
Could something like this work in general? `my_string.encode(:xml => :text)`

[StackOverflow](https://stackoverflow.com/a/7573833)

Reply:
totally - i think we have a couple of options here. my first thought was to test this via a CI script, so I opened this card: https://github.com/openstax/cnx-recipes/issues/4657 

i wonder if surrounding the value text with quotes would also work, though. setting that as a norm (and enforcing it with some kind of linter) would maybe be easier. any thoughts?

Reply:
I was trying to figure out what nokogiri is doing and ended up with [these specs](https://github.com/openstax/cookbook/compare/bye-namespace) to help figure out when the `m:` is being removed from a document.

It seems that if any content is invalid (even is nokogiri has the STRICT flag set) then nokogiri will try to clean it up